docs/index.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: eduMate
```

Comment:
We may need to discuss whether it is still eduMate, rather than EduMate which can uncomplicate the Java elements.

Reply:
Do you want to just open an issue on this?